MainClass.java

```diff
@@ -0,0 +1,34 @@
+package gb.lvl3lesson5;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.CyclicBarrier;
+import java.util.concurrent.Semaphore;
+
+public class MainClass {
+    public static final int CARS_COUNT = 4;
+
+    public static void main(String[] args) throws InterruptedException {
+        CountDownLatch cdlStart = new CountDownLatch(CARS_COUNT);
+        CountDownLatch cdlFinish = new CountDownLatch(CARS_COUNT);
+        CyclicBarrier cyclicBarrier = new CyclicBarrier(CARS_COUNT);
+
+
+        System.out.println("ВАЖНОЕ ОБЪЯВЛЕНИЕ >>> Подготовка!!!");
+        Race race = new Race(new Road(60), new Tunnel(new Semaphore(2)), new Road(40, true));
```

Comment:
```suggestion
        Race race = new Race(new Road(60), new Tunnel(new Semaphore(CARS_COUNT / 2)), new Road(40, true));
```
как более универсальное решение